### PR TITLE
Preparing for v2.7.1

### PR DESCRIPTION
**Pull request by louib**
The goal is to have all the patches upstreamed before 2.7.1 is released.
Depends on https://github.com/keepassxreboot/keepassxc/pull/7728

### CODE REVIEW

**Comment by louib** [comment]
@AsavarTzeth Do you have merge permission on this repo? This one is ready for a review :pray:

File: org.keepassxc.KeePassXC.yml [hunk 1]
```diff
@@ -28,13 +28,17 @@ finish-args:
...
   - --device=all
-    # Access to temporary files (Remove if RuntimeDirectory patch is upstreamed)
-  - --filesystem=/tmp
     # SSH Agent access
   - --socket=ssh-auth
     # Host access: workaround extensive portal issues
     # (use beta branch for testing of portals based file access)
   - --filesystem=host
+    # Access to temporary files (Remove if RuntimeDirectory patch is upstreamed)
+  - --filesystem=/tmp
```
Comment (droidmonkey) [inline org.keepassxc.KeePassXC.yml:37]: Given this comment, should we remove this?

**Comment by droidmonkey** [comment]
@AsavarTzeth would you mind granting us ownership over this repo?

File: org.keepassxc.KeePassXC.yml (hunk 1 above)
Comment (louib) [reply org.keepassxc.KeePassXC.yml:37]: I'm not sure I understand this comment. When do we use the tmp dir?

Comment (droidmonkey) [reply org.keepassxc.KeePassXC.yml:37]: When we save using temporary files moved into place. I just didn't know if we upstreamed the changes mentioned in the comment. If not, please ignore me.

Comment (louib) [reply org.keepassxc.KeePassXC.yml:37]: I don't think we upstreamed changes related to saving to the `/tmp` dir.

File: org.keepassxc.KeePassXC.yml [hunk 2]
```diff
@@ -28,13 +28,17 @@ finish-args:
...
+    # Required for native messaging (Tor browser)
+  - --filesystem=xdg-data
+    # Required for native messaging (Non-Firefox)
+  - --filesystem=xdg-config
```
[the remark this answers is not in the document]
Comment (TingPing) [reply org.keepassxc.KeePassXC.yml:41]: You have `host` already. So everything is already accessible.

File: org.keepassxc.KeePassXC.yml (hunk 1 above)
Comment (TingPing) [reply org.keepassxc.KeePassXC.yml:37]: Its very unlikely you need this. You always have a private `/tmp`, and with recent `flatpak` its shared with all instances of the same app.

So unless you make files in `/tmp` to pass to *other* applications, which isn't great, this isn't needed.

**Comment by droidmonkey** [comment]
Accepted thank you!

File: org.keepassxc.KeePassXC.yml (hunk 1 above)
Comment (droidmonkey) [reply org.keepassxc.KeePassXC.yml:37]: We do use /tmp for storage of attachments that are opened in other applications.

File: org.keepassxc.KeePassXC.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (louib) [reply org.keepassxc.KeePassXC.yml:41]: That's right, but at some point hopefully the host access will be removed, and we will have the granular permissions already defined.

[the remark this answers is not in the document]
Comment (louib) [reply org.keepassxc.KeePassXC.yml]: I commented out those permissions while we have `host` access configured.

**Review by droidmonkey — APPROVED** [review]

**Comment by droidmonkey** [comment]
Merged onto `beta` branch for testing

**Pull request merged**